blog/authors.yml

```diff
@@ -1,23 +1,38 @@
-vimayya:
+mayya:
   name: Vinaya Mayya
   title: Senior Technical Program Manager - API
   url: https://www.ed-fi.org/success-stories/category/vinaya-mayya/
   image_url: https://www.ed-fi.org/assets/2019/11/Vinaya-Mayya-768x768.png
+  page: true
 
 hoekstra:
   name: Jason Hoekstra
   title: Senior Program Manager - Tools
   url: https://www.ed-fi.org/success-stories/category/jason-hoekstra/
   image_url: https://www.ed-fi.org/assets/2019/11/Jason-Hoekstra-2022-768x768.png
+  page: true
 
-sfuqua:
+fuqua:
   name: Stephen A. Fuqua
   title: Director, Software Engineering
   url: https://www.ed-fi.org/success-stories/category/stephen-fuqua/
   image_url: https://www.ed-fi.org/assets/2020/10/Stephen-Fuqua-768x768.png
+  page: true
+  socials:
+    linkedin: stephenfuqua
+    github: stephenfuqua
 
 yilmaz:
   name: Mustafa Yilmaz
   title: Senior Program Manager - Data Standard
   url: https://www.ed-fi.org/success-stories/category/mustafa-yilmaz/
   image_url: https://www.ed-fi.org/assets/2022/08/Mustafa-Yilmaz-July-2022-Headshot-768x768.png
+  page: true
+
+hunter:
+  name: Robert Hunter, Jr
+  title: Software Engineering
+
+ragone:
+  name: Maria Ragone
+  title: Technical Business Analyst
```

Comment:
@roberthunterjr (hm... Maria isn't in here yet...)

We'll add links to your headshots and www landing pages as soon as they are available. Welcome to add social links here if interested.

Reply:
Sounds good I will create a PR that adds this. Head shots are scheduled for the Wednesday following Summit.